sessionctx/variable/session_test.go

```diff
@@ -225,6 +225,7 @@ func (*testSessionSuite) TestSlowLogFormat(c *C) {
 # PD_total: 11
 # Backoff_total: 12
 # Write_sql_response_total: 1
+# Result_set: 12345
```

Comment:
how about calling it result_rows? Result_set is not clear whether it is a row or it is a size.